services/src/main/java/org/fao/geonet/api/records/MetadataSampleApi.java

```diff
@@ -267,6 +267,20 @@ SimpleMetadataProcessingReport addSamples(
                             isTemplate = templateName.startsWith(templateOfSubTemplatePrefix) ?
                                 "t" : "s";
                         }
+                        if (isTemplate.equals("s")) {
+                            // subtemplates loaded here can have a title or uuid attribute
+                            String tryUuid = xml.getAttributeValue("uuid");
+                            if (tryUuid != null && tryUuid.length() > 0) uuid = tryUuid;
+                            title = xml.getAttributeValue("title");
```

Comment:
Would it be better to only extract the title when we index all different types of subtemplate ? like https://github.com/geonetwork/core-geonetwork/blob/master/schemas/iso19139/src/main/plugin/iso19139/index-fields/index-subtemplate.xsl#L97

Reply:
Looking at the old use case for this, indexing xslt is now flexible enough to handle the title generation so we can drop this. Also, other code avoids using or populating database title field so yes it is more consistent to leave this to the indexing xslt.